wsl-instructions.md

```
@@ -21,5 +21,14 @@ sudo update-alternatives --install /usr/bin/clang clang /usr/bin/clang-22 100 \
 --slave /usr/bin/clang++ clang++ /usr/bin/clang++-22
 sudo apt install -y catch22 ccache strace
 ```
-8. In the repo, run `code .`, which brings up VSCode remotely.
-9. Run `./cleanbuild.sh tidy` to build the tests with clang-tidy enabled. You can specify `libstdcpp` or `libcxx` as the first argument to choose the standard library.
+8. Install the latest GCC for its libstdc++.
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)